tests/drivers/rtc/rtc_api/src/test_update_callback.c

```diff
@@ -34,7 +34,11 @@ ZTEST(rtc_api, test_update_callback)
 
 	ret = rtc_update_set_callback(rtc, NULL, NULL);
 
-	zassert_ok(ret, "Failed to clear and disable update callback");
+	if (-ENOSYS == ret) {
```

Comment:
Please use ret == const :) I know its technically less safe, but read easier is :D

Reply:
Ok.